client/src/Main.elm

```diff
@@ -1511,6 +1611,15 @@ update msg model =
         GbNewLevel0Loaded content ->
             ( { model | level0GenbankContent = Just content }, Cmd.none )
 
+        Level0Received (Ok level0s) ->
+            ( { model | insertList = level0s }, Cmd.none )
+
+        Level0Received (Err err) ->
+            ( model, Cmd.none )
+
+        RequestAllLevel0 ->
+            ( model, authenticatedGet (Maybe.withDefault "" model.token) "http://localhost:8000/vectors/level0" Level0Received (Decode.list level0Decoder) )
```

Comment:
Line is too long.

I would be inclined to do something different is model.token is empty. E.g. display an error

```suggestion
            case model.token of
              Just token -> -- blah blah
              Nothing -> -- show error
```

Reply:
OK, Here's my proposition:

```
RequestAllLevel0 ->
            case model.token of
                Just token ->
                    ( model
                    , authenticatedGet (Maybe.withDefault "" model.token)
                        "http://localhost:8000/vectors/level0"
                        Level0Received
                        (Decode.list level0Decoder)
                    )

                Nothing ->
                    ( { model
                        | error =
                            Just <|
                                showHttpError (BadBody "Permission denied! Token is empty!")
                      }
                    , Cmd.none
                    )
```